x-pack/plugins/apm/common/utils/formatters/duration.ts

```diff
@@ -33,11 +33,12 @@ export type TimeFormatter = (
   options?: FormatterOptions
 ) => ConvertedDuration;
 
-type TimeFormatterBuilder = (max: number) => TimeFormatter;
+type TimeFormatterBuilder = (max: number, threshold?: number) => TimeFormatter;
 
-export function getUnitLabelAndConvertedValue(
+function getUnitLabelAndConvertedValue(
   unitKey: DurationTimeUnit,
-  value: number
+  value: number,
+  threshold: number = 10
```

Comment:
super duper nit: I think it would be good to have a comment on what's the purpose of `threshold`, as it's not obvious at first glance.

Reply:
Added in 03fadfe.